src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,50 @@
+package codeu.controller;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+
+public class ProfileServlet extends HttpServlet
+{
+    private User user;
+    //private UserStore userStore;
+
+    //Forwards the request to the profile.jsp to create the profile page
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+    throws IOException, ServletException 
+    {   
+        //Checks to see if the user is logged into his or her page
+        if (request.getSession().getAttribute("user") != null) 
+        {
+            request.getRequestDispatcher("/WEB-INF/view/profile.jsp").forward(request, response);
+            return;
```

Comment:
is it necessary to have a return statement after the request forwarding? since it's just an if-else statement so only one of the clauses would ever be entered, and it looks like from what i can see the method would terminate after that request anyways!